Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -181,10 +182,19 @@ namespace ROS2
             auto collidersNames = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), false, true);
             auto visualNames = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), true, false);
 
+            AZ::Crc32 params_crc;
+            for (auto& [key, value] : m_params)
+            {
+                params_crc.Add(key);
+                params_crc.Add(value);
+            }
```

Comment:
The [AZ::Uuid::CreateName](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Math/Uuid.h#L77-L78) function to create a Uuid from a string.
The [Uuid::operator+](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Math/Uuid.h#L143-L144) can then be used to combine the Uuid of two strings